### ISSUE TITLE
Water Setbacks

### ISSUE THREAD

**Issue by alopez04** [NREL/reVX#147]
**Why this feature is necessary:**
Water setbacks have become more common as an ordinance for wind and pv. Similar to creating setbacks for other features (e.g., transmission, roads), we need one for water features. 

**A possible solution is:**
Use the functionality already created for other feature setbacks. 

**I have considered the following alternatives:**
None.

**Additional context**
None

**Charge code**
WEPG.10348.08.01.06

**Urgency / Timeframe**
Urgent / ASAP

**Comment by grantbuster** [comment]
@ppinchuk i saw you self assigned, are you already working on this? Do you already know the details of what needs to be done?

**Comment by ppinchuk** [comment]
Yes, @grantbuster, most of this is actually done already in [the water setbacks branch](https://github.com/NREL/reVX/tree/pp/water_setbacks), but I haven't submitted a PR because my large-scale tests on eagle have been hanging for several days. Once those are done and look good, I will submit!

**Comment by ppinchuk** [comment]
Ok, I think the [CONUS tests look good](https://github.com/NREL/reVX/files/8950244/water_setbacks.pdf).
